core/eip3529tests/eip3529_ethash_test.go

```diff
@@ -36,42 +32,80 @@ func TestSelfDestructGasPreEIP3529(t *testing.T) {
 	// The refund of 24000 gas (i.e. params.SelfdestructRefundGas) is not applied since refunds pre-EIP3529 are
 	// capped to half of the transaction's gas.
 	expectedGasUsed := (params.TxGas + vm.GasQuickStep + params.ColdAccountAccessCostEIP2929 + params.SelfdestructGasEIP150) / 2
-	testGasUsage(t, preLondonConfig(), ethash.NewFaker(), bytecode, nil, 60_000, expectedGasUsed)
+	TestGasUsage(t, preLondonConfig(), ethash.NewFaker(), bytecode, nil, 60_000, expectedGasUsed)
 }
 
-func TestSstoreGasPreEIP3529(t *testing.T) {
+func TestSstoreModifyGasPreLondon(t *testing.T) {
```

Comment:
This code is already on develop. I don't understand why Github is showing this as new addition.